core/src/main/java/com/linecorp/armeria/common/DefaultCookie.java

```diff
@@ -69,7 +72,9 @@
         this.maxAge = maxAge;
         this.secure = secure;
         this.httpOnly = httpOnly;
+        this.hostOnly = hostOnly;
         this.sameSite = sameSite;
+        createdMillis = System.currentTimeMillis();
```

Comment:
Similarly, it'd be nice if a user can specify the creation time when building a `Cookie`. How about adding methods like `CookieBuilder.createdAt(long)` and `createdAt(Instant)`? If not set, we could use `System.currentTimeMillis()`.

Reply:
Since we're handling expiry in cookie jar, I added a `set` method with a `long createdTimeMillis`.